server/storage/mvcc/watchable_store.go

```diff
@@ -285,9 +285,9 @@ func (s *watchableStore) moveVictims() (moved int) {
 					newVictim = make(watcherBatch)
 				}
 				newVictim[w] = eb
+				moved++
```

Comment:
https://github.com/etcd-io/etcd/blob/28b9089de970c2759c9f3439a0493bd170dbbfe9/server/storage/mvcc/watchable_store.go#L247.

At syncVictimsLoop, it will always loop until `watchableStore.moveVictims() != 0`.
In `moveVictims`, need't always make moved++ at every element in `victims `. Only send event failed, make moved++.